### PR TITLE
add header to response in cli

### LINKED ISSUE #84: Browser Payjoin Senders Blocked via CORS
Allow CORS directly from rouille with payjoin-cli. Just add the Access-Control-Allow-Origin header to the response.

The following gives the desired behavior NGINX

```
proxy_hide_header Access-Control-Allow-Origin;
add_header Access-Control-Allow-Origin * always;
```

### PR DESCRIPTION
**Pull request by jbesraa**
resolves #84

### CODE REVIEW

**Comment by DanGould** [comment]
I think you can leave  `.with_additional_header("Access-Control-Allow-Origin", "*")` in one line on [app.rs:164](https://github.com/payjoin/rust-payjoin/pull/88/files#diff-8e6acd92d1374811117a8f9f08387d1a66bd8a3d2ae69c669dd65bab00e441e5R164) instead of on each response

File: payjoin-cli/src/app.rs [hunk 1]
```diff
@@ -477,4 +487,6 @@ impl payjoin::receive::Headers for Headers<'_> {
     }
 }
 
-fn serialize_psbt(psbt: &Psbt) -> String { base64::encode(&psbt.serialize()) }
+fn serialize_psbt(psbt: &Psbt) -> String {
```
Comment (DanGould) [inline payjoin-cli/src/app.rs:490]: in general I'd like to see formatting in a separate commit to semantic changes.

This crate is formatted with nightly, `cargo +nightly fmt`, which this PR appears to fail.

**Review by DanGould — CHANGES_REQUESTED** [review]
Awesome to have this contribution. I left 2 concerns. Thank you!

**Comment by jbesraa** [comment]
@DanGould moved to the sever initialization. thanks for pointing this out. 
I added the header to local-http as well although iam not sure its strictly needed(?). but could be better for consistency and testing?

**Comment by DanGould** [comment]
Since this is added in both spots after `handle_web_request` I think this can be fixed by changing that function with one line.

**Review by DanGould — APPROVED** [review]

**Pull request merged**